playground/cli/jsonToPDF/twit_thread.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "name": "Naval",
+    "twitterHandle": "naval",
+    "image": "https://pbs.twimg.com/profile_images/1256841238298292232/ycqwaMI2_400x400.jpg",
+    "createAt": "2018-05-31",
```

Comment:
property `createAt` is converted to `createdAt` in pr #351 , can you also change them in your own code, to maintain a standart data.